documentation/resource_plist.md

```diff
@@ -2,7 +2,7 @@ plist
 =====
 
 Use the **plist** resource to manage property list files (plists) and their content.
-A **plist** resource instance represents the state of a single key-value pair in
+The [**plist**](https://github.com/Microsoft/macos-cookbook/blob/master/resources/plist.rb) resource instance represents the state of a single key-value pair in
```

Comment:
Again, use of the word "instance" in this sentence is important. _Many_ instances of _a_ `plist` resource can be used within one resource collection. It doesn't make sense to have "the" everywhere we talk about our custom resources.

Reply:
```suggestion
The [**plist**](https://github.com/Microsoft/macos-cookbook/blob/master/resources/plist.rb) resource manages the state
```